app/controllers/dispatch.rb

```diff
@@ -0,0 +1,106 @@
+get '/mission' do
```

Comment:
if you have a resource called `Mission`, then the RESTful routing convention to show all the missions in your database would be a `GET` request to `/missions` - notice how mission is pluralized since we are talking about the collection of missions.